back/engines/commercial/machine_translations/spec/services/machine_translations/machine_translation_service_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe MachineTranslations::MachineTranslationService do
+  let(:service) { described_class.new }
```

Comment:
```suggestion
  subject { described_class.new }
```
and of course `subject` replaces `service` further down.

Using `subject` in RSpec is important. For this test, it is even not necessary because `subject { described_class.new }` is the default in RSpec, so you can remove the whole line.